versions/run.noaacloud.ver

```diff
@@ -5,4 +5,8 @@ export spack_env=gsi-addon-env
 source "${HOMEgfs:-}/versions/spack.ver"
 export spack_mod_path="/contrib/spack-stack/spack-stack-${spack_stack_ver}/envs/gsi-addon-env/install/modulefiles/Core"
```

Comment:
Do you still need this run version file (`run.noaacloud.ver`) if you have the other two now? If so, when will this one be used?